projects/adapter/src/dbt/adapters/fal_experimental/utils/environments.py

```diff
@@ -238,13 +238,20 @@ def _parse_remote_config(
         "target_environments": [env_definition],
     }
 
+def _get_package_from_type(adapter_type: str):
+    SPECIAL_ADAPTERS = {
+        # Documented in dbt website
```

Comment:
nice call-out, maybe also leave the link?

Reply:
done!